raiden/transfer/node.py

```diff
@@ -745,6 +746,14 @@ def handle_receive_unlock(
     return subdispatch_to_paymenttask(chain_state, state_change, secrethash)
 
 
+def handle_update_transport_timestamp(
+        chain_state: ChainState,
+        state_change: StateChange,
```

Comment:
The type of the state change here should be more specific. `ActionUpdateTransportTimestamp`